### PR TITLE
improved terminal 

### PR DESCRIPTION
**Pull request by hrmnpfg**
added a dependency on jline for better terminal use

### CODE REVIEW

**Comment by kinoko-neko** [comment]
jline??? you mean jeans line?

**Review by Acors24 — APPROVED** [review]

File: Makefile
Comment (hrmnpfg) [inline Makefile:1]: sad

Comment (kinoko-neko) [reply Makefile:1]: makefile fixes itself? wdym?

**Review by kinoko-neko — APPROVED** [review]
baza

**Pull request merged**